Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: tests doc build
 
-lint:	
-	flake8 . --exclude=doc
+lint:
+	flake8 . --exclude=doc,.venv
```

Comment:
Have you encountered a problem that justifies this addition to the file?

Reply:
Hello @thibaultcordier. By default the `lint` command in your Makefile will scan every folder in the current workspace. `.venv` is the default location when using vscode and virtual envs, so your `lint` command was also scanning every dependency in the virtual env folder.

Reply:
Hey @pidefrem,
Could you please remove the environment? You can also save the env somewhere else than in the MAPIE folder. 
Thank you!